src/platform/macos.rs

```diff
@@ -140,6 +161,10 @@ pub trait WindowBuilderExtMacOS {
     fn with_has_shadow(self, has_shadow: bool) -> WindowBuilder;
     /// Window accepts click-through mouse events.
     fn with_accepts_first_mouse(self, accepts_first_mouse: bool) -> WindowBuilder;
+    /// Set whether the `OptionAsAlt` key is interpreted as the `Alt` modifier.
```

Comment:
```suggestion
    /// Set how the <kbd>Option</kbd> keys are interpreted as the `Alt` modifier.
```

Reply:
Yeah, I just reverted the bits used before + synchronized the handling of `key_up`/`key_down`.